.github/ISSUE_TEMPLATE/documentation-update.yaml

```diff
@@ -0,0 +1,17 @@
+---
+name: Documentation request
+about: Use this to propose information improvements and additions
+title: "[ Documentation request ] "
+labels: ''
+assignees: ''
+
+---
+
+# Brief description
+
+Provide a description of your documentation request.
```

Comment:
Please add attribute, types, label, description and required field. You can take inspiration from https://github.com/keploy/keploy/blob/main/.github/ISSUE_TEMPLATE/--feature-request.yaml